admin/server/billing.go

```diff
@@ -193,12 +194,26 @@ func (s *Server) UpdateBillingSubscription(ctx context.Context, req *adminv1.Upd
 
 	if planChange {
 		// send plan changed email
-		err = s.admin.Email.SendPlanUpdate(&email.PlanUpdate{
-			ToEmail:  org.BillingEmail,
-			ToName:   org.Name,
-			OrgName:  org.Name,
-			PlanName: plan.DisplayName,
-		})
+
+		if plan.PlanType == billing.TeamPlanType {
+			// special handling for team plan to send custom email
+			err = s.admin.Email.SendTeamPlanStarted(&email.TeamPlan{
+				ToEmail:          org.BillingEmail,
+				ToName:           org.Name,
+				OrgName:          org.Name,
+				FrontendURL:      s.admin.URLs.Frontend(),
+				PlanName:         plan.DisplayName,
+				BillingStartDate: sub.CurrentBillingCycleStartDate, // TODO: should this be end date + 1?
```

Comment:
depends on what we want if this changed at any day we want to show current day or day+1 @ericokuma can confirm ?

Reply:
Updated to use end date.